bin/exec.go

```diff
@@ -2,13 +2,14 @@ package main
 
 import (
 	"flag"
+	log "github.com/sirupsen/logrus"
```

Comment:
Unable to change this sequence

Reply:
This will be like:
```
	"flag"
	"time"

	log "github.com/sirupsen/logrus"
	"k8s.io/client-go/rest"
	"k8s.io/client-go/tools/clientcmd"

	"github.com/litmuschaos/chaos-executor/pkg/utils"
	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
```